src/libs/TransactionUtils.ts

```diff
@@ -330,6 +330,16 @@ function isExpensifyCardTransaction(transaction: Transaction): boolean {
     return isExpensifyCard(transaction.cardID);
 }
 
+/**
+ * Determine whether a transaction is made with a card.
+ */
+function isCardTransaction(transaction: Transaction): boolean {
+    if (!transaction.cardID) {
+        return false;
+    }
+    return transaction.cardID > 0;
```

Comment:
+1

Reply:
<img width="442" alt="image" src="https://github.com/Expensify/App/assets/38015950/db2f7c2b-74bb-4f21-ad46-a92f63b131e4">
Even though this is correct, I end up with this TS problem because cardID can be undefined

Reply:
I can do this instead to get around TS problems, but I feel like it's also fine as is 
<img width="435" alt="image" src="https://github.com/Expensify/App/assets/38015950/991f9b18-df53-4e6b-94e4-48f030e121a0">

Reply:
@akinwale

Reply:
@grgia Thanks. I think I prefer the second option, and you could also make it a one-liner: `return (transaction?.cardID ?? 0) > 0` (no strong preference here).

I still haven't been able to test the functionality in the app as stated here: https://github.com/Expensify/App/pull/31353#issuecomment-1816690945. Any ideas on how I can test this?